src/V3Width.cpp

```diff
@@ -2866,6 +2867,10 @@ class WidthVisitor : public AstNVisitor {
     void methodCallClass(AstMethodCall* nodep, AstClassRefDType* adtypep) {
         // No need to width-resolve the class, as it was done when we did the child
         AstClass* first_classp = adtypep->classp();
+        if (nodep->name() == "randomize") {
```

Comment:
Probably need a test and error (I think it's illegal please check) if the user declares a class "randomize" function of their own.

Probably need a test and unsupported message for pre_randomize/post_randomize.